templates/base.html

```diff
@@ -1,28 +1,10 @@
 {% load static %}
 
 <head>
-  
-<!-- CSS -->
-<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/css/bootstrap.min.css" integrity="sha384-Smlep5jCw/wG7hdkwQ/Z5nLIefveQRIY9nfy6xoR1uRYBtpZgI6339F5dgvm/e9B" crossorigin="anonymous">
-
-<link href="https://fonts.googleapis.com/icon?family=Material+Icons"
-      rel="stylesheet">
-
-<link href="https://fonts.googleapis.com/css?family=Montserrat:300,700" rel="stylesheet">
-
- <link rel="stylesheet" href="https://unpkg.com/ngx-bootstrap/datepicker/bs-datepicker.css">
-
-
 
 <!-- Icons -->
 <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.1/css/all.css" integrity="sha384-O8whS3fhG2OnA5Kas0Y9l3cfpmYjapjI0E4theH4iuMD+pLhbf6JI0jIMfYcK3yZ" crossorigin="anonymous">
```

Comment:
@Chibi-Shem use the angular version for this package. All our packages should be included in the bundle files. 
https://www.npmjs.com/package/angular-font-awesome